lib/iex/resources/chart.rb

```diff
@@ -1,28 +1,27 @@
+require_relative 'base'
 require_relative 'chart/base'
```

Comment:
What's your thoughts about the lines above? It is clear that they are distinct base classes but I don't know about having the same name. If you see bellow, I'm calling Base.symbol passing the full path so there is no conflict. Should we keep it?

Reply:
It's totally fine.